examples/google-secure-signals-integration/with_sdk_v3/public/ads.js

```diff
@@ -175,7 +175,6 @@ function onAdEvent(adEvent) {
 function onAdError(adErrorEvent) {
   // Handle the error logging.
   console.log(adErrorEvent.getError());
-  adsManager.destroy();
```

Comment:
Google doesn't seem to have the .destroy function any more, and this is only on ad error anyway so I don't think it matters much.